tools/cli/workflow.go

```diff
@@ -143,6 +143,11 @@ func newWorkflowCommands() []cli.Command {
 					Usage: "Optional input for the workflow from JSON file. If there are multiple JSON, concatenate them and separate by space or newline. " +
 						"Input from file will be overwrite by input from command line",
 				},
+				cli.StringFlag{
```

Comment:
Maybe worth using IntFlag, and use IsSet() to tell difference between 0 and isSet.

Reply:
Thanks. This is what I am looking for. Updated